### DISCUSSION TITLE
1A. Population Statistics - Differences between Namibia and South Africa After Independence

### DISCUSSION THREAD

**Discussion by angiechen17** [Chris-Nicholas-collaborators/EEP153_team_Malthus#6] [General]
Years: pre and post-1990 (year of Namibia's independence from South Africa)
Countries: Namibia, South Africa
People: all
Ages: all

**Comment by t-pappas** [comment]
ethnic group comparisons between south africa and namibia

**Comment by samlwang2** [comment]
cost of living
